packages/desktop/package.json

```diff
@@ -84,7 +84,7 @@
     "lint": "eslint --ext .jsx,.js,.ts,.tsx ./src/ --fix",
     "lint-ci": "eslint --ext .jsx,.js,.ts,.tsx ./src/",
     "type-check": "tsc -p tsconfig.build.json --noEmit",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
I had to do this one, because with tests running concurrently it was hammering my laptop to the point where even touchpad was not responsive